src/Forms/Shared/CategoryListPage.xaml.cs

```diff
@@ -44,6 +48,23 @@ private void Initialize()
             BindingContext = ViewModel;
         }
 
+        private async void CheckForKey(object sender, EventArgs e)
+        {
+            this.Appearing -= CheckForKey;
+
+            await ApiKeyManager.TrySetLocalKey();
```

Comment:
Could this throw an exception? If so, it might not be handled?

Reply:
It is not possible for `TrySetLocalKey` to throw an exception. The potential exceptions are handled in that method.